regression/python/tests.mk

```diff
@@ -46,7 +48,16 @@ DEFINES += -D_XOPEN_SOURCE=500 -D_BSD_SOURCE
 CDEFINES   += $(DEFINES)
 CXXDEFINES += $(DEFINES)
 
-FLAGS     = -g -Wall $(shell python3.6-config --cflags) -O1
+# To check if we are in an Anaconda python environment
+CONDA_PREFIX := $(shell echo $$CONDA_PREFIX)
+
+ifeq ($(CONDA_PREFIX),)
+  FLAGS     = -g -Wall $(shell python3-config --cflags) -O1 
+  LDFLAGS  += $(shell python3-config --ldflags)
+else
+  FLAGS     = -g -Wall -I$(CONDA_PREFIX)/include/python3.7m -O1 
+  LDFLAGS  += -L$(CONDA_PREFIX)/lib -lpython3.7m -Wl,-rpath,$(CONDA_PREFIX)/lib
+endif
+
```

Comment:
Is it worth it to have a separate regression directory for pytorch?

Reply:
This part is not pytorch specific. This logic is just to distinguish between system python and anaconda's python...

Reply:
I think there is value in splitting the pytorch regression tests into a separate regression directory. They depend on the anaconda environment, right?

Reply:
Definitely, I also have been thinking that having separate directories would make everything easy to understand. Sure, I'll add a separate pytorch regression directory.